api/openapi.yaml

```diff
@@ -823,6 +833,16 @@ components:
             The verbose mode to use in the call to the ansible-playbook command. 1 = -v,
             2 = -vv, etc. Valid values range from 0 to 4. See the ansible-playbook help
             for more information.
+        ansiblePassthrough:
+          type: string
+          pattern: '^[\w\-_=, ]*$'
+          description: >-
+            Additional parameters that are added to all Ansible calls for the session. WARNING:
```

Comment:
I think this docstring should enumerate the passthrough options that are supported and say that anything else is an error.

Reply:
updated.